examples/model_selection/plot_cost_sensitive_learning.py

```diff
@@ -0,0 +1,752 @@
+"""
+===============================================================
+Post-tuning decision threshold based on cost-sensitive learning
+===============================================================
+
+Once a classifier is trained, the output of the :term:`predict` method output class
```

Comment:
Once a classifier is trained, the output of the :term:`predict` method output**s** class